daemons/gptp/common/avbts_clock.hpp

```diff
@@ -44,11 +44,23 @@
 
 #define EVENT_TIMER_GRANULARITY 5000000		/*!< Event timer granularity*/
 
-#define INTEGRAL 0.0024				/*!< PI controller integral factor*/
+#define INTEGRAL 0.0003				/*!< PI controller integral factor*/
 #define PROPORTIONAL 1.0			/*!< PI controller proportional factor*/
 #define UPPER_FREQ_LIMIT  250.0		/*!< Upper frequency limit */
 #define LOWER_FREQ_LIMIT -250.0		/*!< Lower frequency limit */
 
+#define UPPER_LIMIT_PPM 250
```

Comment:
I guess that these new macros apply just to the syntonization mode, so it might be helpful to have a comment saying as much if that's really the case.